.github/workflows/install.sh

```diff
@@ -22,6 +22,7 @@ case $MODE in
 compile|test|clean)
```

Comment:
This file vanished in #670
Can you rebase and adapt ?

Reply:
If you sync to head, then hopefully also #678 will make sure that the Kokoro test will run through.

Reply:
Rebased. Kokoro still fails and I'm unable to download `test.log` ("File not found") [on artefacts subpage](https://source.cloud.google.com/results/invocations/b37e98ae-0022-42e8-99e6-abf55883a751/artifacts/bazel_test_logs%2Fbazel-out%2Fk8-fastbuild%2Ftestlogs%2Fverilog%2Ftools%2Fsyntax%2Fexport_json_examples%2Fverible_verilog_syntax_py_test)

I guess `sudo ln -sf /usr/bin/python3.8 /usr/bin/python3` is needed (I had to do it for GH Actions)

Reply:
The symlink breaks "add-apt-repository". Some possible solutions:
* Don't use `add-apt-repository` and add repositories manually instead ([example](https://github.com/google/verible/blob/bd5fc261a9d26f18bb8240eef832c1d4db700087/.github/workflows/install-python.sh#L18))
* Create symlink just before running bazel (after all other `steps/*` scripts)
* Try [renaming apt_pkg](https://askubuntu.com/questions/1090079/what-problems-can-i-expect-using-deadsnakes-ppa-as-my-primary-python3-instance-t) (hack)